infra/resources/modules/commons/apim.tf

```diff
@@ -294,3 +294,40 @@ resource "azurerm_api_management_subscription" "support" {
   state               = "active"
   allow_tracing       = false
 }
+
+####################################################################################
+# BACKUP
+####################################################################################
+
+module "storage_account" {
+  source = "github.com/pagopa/dx//infra/modules/azure_storage_account?ref=main"
+
+  environment = {
+    prefix          = var.prefix
+    env_short       = var.env_short
+    location        = var.location
+    app_name        = "backup"
+    instance_number = "01"
+  }
+
+  tier                = "s"
+  resource_group_name = azurerm_resource_group.rg_routing.name
+
+  subnet_pep_id                        = module.pendpoints_snet.id
+  private_dns_zone_resource_group_name = azurerm_resource_group.net_rg.name
+
+  subservices_enabled = {
+    blob  = true
+    file  = false
+    queue = false
+    table = false
+  }
+
+  tags = var.tags
+}
+
+resource "azurerm_storage_container" "backup" {
+  name                  = "apim-backup"
+  storage_account_name  = module.storage_account.name
```

Comment:
Deprecated in favor of [storage_account_id](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/storage_container#storage_account_id-1)

Reply:
it has been deprecated since 4.9.0, here 3.108 is used